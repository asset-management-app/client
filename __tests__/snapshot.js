import { render } from '@testing-library/react';
import Index from '../src/app/page';

jest.mock('next/navigation', () => ({
  redirect: jest.fn(),
}));

it('renders homepage unchanged', () => {
  const { asFragment } = render(<Index />);
  expect(asFragment()).toMatchSnapshot();
});