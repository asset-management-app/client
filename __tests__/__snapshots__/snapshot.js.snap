// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<DocumentFragment>
  <h1>
    404 Not Found
  </h1>
</DocumentFragment>
`;
